TM1py/Services/CubeService.py

```diff
@@ -189,6 +189,40 @@ def get_dimension_names(self, cube_name: str, skip_sandbox_dimension: bool = Tru
             return dimension_names[1:]
         return dimension_names
 
+    def get_names_with_dimension(self, dimension_name: str, model_cubes_only: bool = False,
+                                 **kwargs) -> List[str]:
+        """ Ask TM1 Server for list of cube names that contain specific dimension
+
+        :param dimension_name: string, valid dimension name (case insensitive)
+        :param model_cubes_only: bool, True will filter result to model cubes only
+        """
+        cube_prefix = 'Cubes' if model_cubes_only == False else 'ModelCubes()'
+        url = format_url(
+            "/api/v1/{}?$select=Name&$filter=Dimensions/any(d: toupper(d/Name) eq toupper('{}'))",
+            cube_prefix, dimension_name
+            )
+        response = self._rest.GET(url, **kwargs)
+        cubes = list(entry['Name'] for entry in response.json()['value'])
+        return cubes
+
+
+    def get_names_with_dimension_wildcard(self, wildcard: str, model_cubes_only: bool = False,
```

Comment:
Does `get_cube_names_with_dimension_wildcard` sound better?

Reply:
similar to above, felt that "cube" was implied

Reply:
I also kinda think it's implied